qgisprocess_user_2024.Rmd

```diff
@@ -194,7 +194,7 @@ layout: true
 library(tmap)
 tm_shape(spData::seine) + 
   tm_lines() + 
-  tm_style("beaver") + 
+  tm_style("albatross_v3") + 
```

Comment:
I think you created the `albatross_v3` style yourself?

`! style "albatross_v3" unknown. The available styles are: "white", "gray", "natural", "cobalt", "col_blind", "albatross", "beaver", "bw", "classic", "watercolor" `

Anyway I tried `albatross` but it wasn't suited for these maps out of the box. For now leaving as-is.

Reply:
I think I'm on a different (dev) version of tmap, but not sure...

Reply:
Well that's the weird part: we have the same version (hence I assumed you crafted this style). Your renv.lock [says](https://github.com/Robinlovelace/user-2024-qgisprocess/blob/6d2a8ce093c70a8a96c620338d7c356ef2163fe9/renv.lock#L1329-L1333):

```
    "tmap": {
      "Package": "tmap",
      "Version": "3.3-4",
      "Source": "Repository",
      "Repository": "CRAN",
```

(and the hash is identical)

Reply:
My guess: I didn't restart R as I should have done..